searx/metrics/__init__.py

```diff
@@ -157,7 +157,11 @@ def get_reliabilities(engline_name_list, checker_results):
 
 
 def round_or_none(number, digits):
-    return round(number, digits) if number else number
+    '''return None if number is None
+    return 0 if number is 0
+    otherwise round number with "digits numbers.
+    '''
+    return round(number, digits) if number is not None else number
 
 
 def get_engines_stats(engine_name_list):
```

Comment:
This function was a little bit hard for me to read, I send a patch trying to make it more readable.